Mission 3-3. Main 댓글 기능

azure928 opened the pull request.

azure928 committed:
branch 생성

azure928 committed:
test

azure928 committed:
Add: 로그인 페이지 마크다운 및 로그인 유효성 검사
- 로그인 페이지 마크다운
- 로그인 유효성 검사

azure928 committed:
test

azure928 committed:
Add: 로그인 페이지, 메인 페이지 마크업
- 기존에 html, css로 만든 마크업을 jsx, scss로 변환
Merge branch 'main' into feature/jiYook

azure928 committed:
Fix: scss 수정
main.scss에 Nesting 적용함

Choi-jeonghoon reviewed (changes requested):
충돌 확인해서 다시한번 pr요청 부탁드립니다.

Choi-jeonghoon commented on src/index.js:
```diff
@@ -1,8 +1,8 @@
+import Router from './Router';
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+const root = ReactDOM.createRoot(document.getElementById('root'));
```
충돌해결부탁드립니다.

Choi-jeonghoon commented on src/pages/jiYook/Login/Login.scss:
```diff
@@ -0,0 +1,62 @@
+}
+
+.login-container {
+  //top: 50%;
```
불필요한 주석은 삭제 부탁드립니다.

Choi-jeonghoon commented on src/pages/jiYook/Main/Main.scss:
```diff
@@ -0,0 +1,223 @@
+    flex-direction: row;
+    width: 935px;
+
+    /*background-color: violet;*/
```
불필요한 주석은 삭제 부탁드립니다.

Choi-jeonghoon commented on src/pages/jiYook/Main/Main.scss:
```diff
@@ -0,0 +1,223 @@
+    }
+  }
+  .main-right {
+    //display: flex;
```
불필요한 주석은 삭제 부탁드립니다.

azure928 committed:
Fix: scss 수정
main.scss에 Nesting 적용함

azure928 committed:
[Add] Main 댓글 기능
- 사용자가 댓글 입력 후 버튼 클릭 시 댓글이 추가되도록 구현
- 아직 구현중임
- main 브랜치 pull 받기위해 중간에 commit 함

azure928 committed:
Add: Mission 3-3. Main 댓글 기능
- 사용자가 댓글 입력 후 enter 를 누르거나 왼쪽의 버튼 클릭 시 댓글이 추가되도록 구현

azure928 committed:
Remove: 불필요한 주석 삭제
- Login.scss, Main.scss 불필요한 주석 삭제

ssujinc commented on src/pages/jiYook/Login/Login.scss:
```diff
@@ -0,0 +1,60 @@
+p,
+button {
+  margin: 0;
+}
```
공통으로 들어갈수있는 css 입니다. 
다른 식으로 사용해주세요.

ssujinc commented on src/pages/jiYook/Main/Main.scss:
```diff
@@ -0,0 +1,218 @@
+p,
+button {
+  margin: 0;
+}
```
공통으로 들어갈수있는 css 입니다. 
다른 식으로 사용해주세요

ssujinc reviewed (commented):
Login.scss 와 Main.scss 의 코멘트 확인해주세요
body style 은 공통으로 적용이 돼서, 다시 확인 부탁드립니다.
